electronicparsers/fhiaims/parser.py

```diff
@@ -2198,8 +2199,22 @@ def _prep_elemental_tier(basis_settings: MSection) -> dict[str, Any]:
                         'Error setting controlIn metainfo.', data=dict(key=key)
                     )
             elif key == 'occupation_type':
-                sec_method.x_fhi_aims_controlIn_occupation_type = val[0]
-                sec_method.x_fhi_aims_controlIn_occupation_width = val[1]
+                sec_smearing = Smearing()
+                sec_electronic.smearing = sec_smearing
+
+                occupation_types_dict = {
+                    'gaussian': 'gaussian',
+                    'methfessel-paxton': 'methfessel-paxton',
+                    'fermi': 'fermi',
+                    'integer': None,
```

Comment:
Please add a comment "integer and cubic smearing don't match well the upstream metainfo definitions "